src/app/update-card-modal/update-card-modal.html

```diff
@@ -0,0 +1,89 @@
+<biggive-popup-standalone *ngIf="card">
+<!--
+  *ngIf="card" above shouldn't be needed - we don't load the modal at all without a card, but it makes the `it('should create'`
+  test pass. Would be good to remove.
+ -->
+    <h2 mat-dialog-title>Update Payment Card </h2>
+
+  <mat-dialog-content class="mat-typography">
+      <form [formGroup]="updateCardForm">
+        <mat-form-field color="primary">
+          <mat-label for="expiryDate">Brand</mat-label>
+          <input
+            [disabled]=true
+            width="5"
+            matInput
+            type="text"
+            title="Brand"
+            [value]="card.brand.toUpperCase()"
+            id="brand"
+            formControlName="last4"
+          >
+        </mat-form-field>
+
+        <mat-form-field color="primary">
+          <mat-label for="expiryDate">Last Four Digits</mat-label>
+          <input
+            [disabled]=true
+            width="5"
+            matInput
+            type="text"
+            title="Last four digits of long number"
+            [value]="card.last4"
+            id="last4"
+            formControlName="last4"
+          >
+        </mat-form-field>
+
+          <mat-form-field color="primary">
+            <mat-label for="expiryDate">Expiry (MM/YY) </mat-label>
+            <input
+              width="5"
+              matInput
+              type="text"
+              title="Expiry date as month and year, e.g. 12/23"
+              pattern="[0-9]{2}/[0-9]{2}"
+              [value]="formattedCardExpiry"
+              id="expiryDate"
+              formControlName="expiryDate"
+            >
+          </mat-form-field>
+
+        <mat-form-field color="primary">
+          <mat-label for="billingAddress">Billing Address</mat-label>
+
+          <!--
+          @todo Prefil with old billing address if possible, work out what format we should present the address in.
```

Comment:
I think that as we discussed last week, postcode and country is probably sufficient for how we currently send + use this? Sounds like that's what we're going to split up from discussion today.

Reply:
Yes that's the plan.